### DISCUSSION TITLE
[Feature request]: Best way to play Vertical and Horizontal Arcade games

### DISCUSSION THREAD

**Discussion by soundsnow** [project64/project64#2345] [General]
### Request box

I just want to get my thoughts across to emulator developers I know.  If you are not interested in this, you can close this issue.  I think there must be quite a few people who need this.

Aspect Ratio
4:3, 16:9 to 3:4, 9:16

Game Category:
Fighter / Vertical
Shooter / Driving Diagonal
Shooter / Driving Horizontal
Shooter / Driving Vertical
Shooter / Flying Diagonal
Shooter / Flying Horizontal
Shooter / Flying Vertical
Shooter / Misc. Horizontal
Shooter / Misc. Vertical

Vertical to Horizontal
Horizontal to Vertical

Device Orientation:
Auto
Portrait, Reverse Portrait
Landscape, Reverse Landscape

Screen / D-Pad Rotation:
Screen: 0 (None), 90° (CW 90°), 180°, 270° (CCW 90°)
D-Pad: 0 (None), 90° (CW 90°), 180°, 270° (CCW 90°)

Screen + D-Pad Rotation:
0 (None), 90° (CW 90°), 180°, 270° (CCW 90°)

Code file size:
[0,90°,180°,270°] 20B
[0,90°,180°,-90°] 20B
[None,CW 90°,180°,CCW 90°] 29B
[Standard,90° Right,Upside Down,90° Left] 43B

### Reason box

I know I've posted an issue, but I didn't explain clearly.

### Examples

![image72](https://user-images.githubusercontent.com/109885044/219587163-d710be70-4875-417e-a258-578052fa2bbd.png)

**Comment by project64** [comment]
changing the video plugin is very low on my todo, so unlikely to get done. But feel free to chat about it.
